### DISCUSSION TITLE
Board view query/display only child notes within the same levels

### DISCUSSION THREAD

**Discussion by nicollasricas** [mabeyj/trilium-collection-views#21] [Q&A] [answered]
Hey @mabeyj,

Thanks for creating this plugin. It's been fun and helpful to use it.

I've been creating a To-Do plugin since I have multiple projects and was hoping to use your views to enhance the user experience.

![image](https://user-images.githubusercontent.com/7860985/171756993-d59cbd71-f4b5-466f-9091-006f6290efaa.png)

As you can see in the image, it's displaying child notes from others projects.

**Is there a way to group by only the child's notes respecting the levels?**

Here's how I have defined the LOCATION label, which I use to group and sort:

The ARCHIVE, BACKLOG, DONE, and IN PROGRESS (aka.: category) have the #sorteableTitle.

![image](https://user-images.githubusercontent.com/7860985/171757440-55dd2d6b-4f61-46bc-9ca8-98fe2969551e.png)

And each To-Do item has a "location" relationship to its parent (which I called a category earlier).

![image](https://user-images.githubusercontent.com/7860985/171757768-7fc3ebad-713b-483b-b526-4d8a52fecb8d.png)

I've checked each location relationship, and NO, they are not relating the category from the other project.

**Comment by mabeyj** [comment, marked as answer]
Hi @nicollasricas, I think this can be done by adding an ancestor condition to `#query`.

> [From Trilium's wiki:](https://github.com/zadam/trilium/wiki/Search#advanced-use-cases)
> ![image](https://user-images.githubusercontent.com/773401/171770323-5a2643d7-2ca2-4bae-8ffa-ada8bf6bbeee.png)

In your first screenshot, you have `#query="#todoItem"` which would return all notes in your entire tree (across all projects) having the `#todoItem` label.

`#query="#todoItem and note.ancestors.title = 'Project 1'"` would return all notes underneath "Project 1" having `#todoItem`.

Since the Project notes are the render notes that render the views, there is a special variable `$title` that is replaced with the render note's title. Then you wouldn't need to hardcode the project name in the query: `#query="#todoItem and note.ancestors.title = $title"` (This also allows you to create a "project board" template, so you could have all of your project notes inherit the same view configuration).

I've roughly recreated your setup, and I think this accomplishes what you're looking for. Project 1 only shows notes under Project 1, etc.:

![image](https://user-images.githubusercontent.com/773401/171772421-b802a2e9-abfc-4f6a-9173-441511c7240c.png)
![image](https://user-images.githubusercontent.com/773401/171772461-7542967b-899c-4435-8043-96f5579f02f9.png)

**Reply by nicollasricas** [reply]
It's working now.

Thanks!
